ocaml/tests/test_certificates.ml

```diff
@@ -216,6 +223,23 @@ let invalid_chain_cert_tests =
     "Validation of an unsupported certificate chain", `Quick, test_cert)
   corrupt_chain_certificates
 
+let hashables =
+  [ "foobar",
```

Comment:
Can we find a better name?

Reply:
any string will do, I could change it to "hashable", but that won't add any information.

Reply:
`__LOC__`, maybe?

Reply:
I think that would make changing this file unnecessarily difficult